pages/test_analytics/ruby_collectors.md

```diff
@@ -32,7 +32,7 @@ Before you start, make sure RSpec runs with access to [CI environment variables]
     bundle
     ```
 
-3. Add the Test Analytics code to your application in `spec/spec_helper.rb`, and set the BUILDKITE_ANALYTICS_TOKEN [securely](/docs/pipelines/secrets) on your agent or agents.
+3. Add the Test Analytics code to your application in `spec/spec_helper.rb`, and set the BUILDKITE_ANALYTICS_TOKEN [securely](/docs/pipelines/secrets) on your agent or agents. Please ensure gems that patch `Net::HTTP` like `httplog` and `sniffer` are required before `buildkite/test_collector` to avoid conflicts.
```

Comment:
Are there links we could provide to the httplog and sniffer gems? That would help confirm what we're talking about for people.

There are also a couple of commas missing:

```suggestion
3. Add the Test Analytics code to your application in `spec/spec_helper.rb`, and set the BUILDKITE_ANALYTICS_TOKEN [securely](/docs/pipelines/secrets) on your agent or agents. Please ensure gems that patch `Net::HTTP`, like `httplog` and `sniffer`, are required before `buildkite/test_collector` to avoid conflicts.
```

Reply:
If there are links we can include, I'd recommend using them and removing the inline code formatting for the gem names.

Reply:
Thanks for the feedback, I've made changes to include your suggestions! 😄